web-registry/src/components/templates/ProjectDetails/ProjectDetails.tsx

```diff
@@ -61,6 +62,15 @@ function ProjectDetails(): JSX.Element {
   const gettingStartedResourcesSection =
     sanityProjectPageData?.allProjectPage?.[0]?.gettingStartedResourcesSection;
 
+  const { data: sanitySoldOutProjects } = useAllSoldOutProjectsQuery({
+    client: sanityClient,
+  });
+
+  const soldOutProjectsIds =
+    sanitySoldOutProjects?.allSoldOutProjects?.[0]?.soldOutProjectsList?.map(
+      project => String(project?.projectId),
+    ) ?? [];
```

Comment:
maybe we can extract this part into a hook that could be re-used in `ProjectCardsSection` too?

Reply:
why aren't we using react-query here?

Reply:
I actually created the query and everything and up not using it 😅, thanks for catching that.